patrick-angelos/Enigma
# Enigma

This project is about a Desktop app in which you can read and publish articles about science.
- A visitor can see:
   - A page with the best article
   - All the latest articles
   - All articles in each category
- A User can:
   - Create an article
   - Upvote and Downvote (Previously upvoted) articles
   - See all articles written by another user

Rails 6.1.3
Ruby 2.7.2
